rpm/qm.spec

```diff
@@ -47,7 +47,8 @@ URL: https://github.com/containers/qm
 Summary: Containerized environment for running Quality Management software
 Source0: %{url}/archive/v%{version}.tar.gz
 BuildArch: noarch
-BuildRequires: %{_bindir}/go-md2man
+# golang-github-cpuguy83-md2man on CentOS Stream 9 is available in CRB repository
```

Comment:
What's CRB repo @sandrobonazzola ? Googled it but couldn't find info.

Reply:
It's https://mirror.stream.centos.org/9-stream/CRB/ , equivalent to CodeReady Builder (https://developers.redhat.com/blog/2018/11/15/introducing-codeready-linux-builder)

Reply:
@sandrobonazzola could you please help us and update our README.md giving instructions to users? I am sure not all users will read the code to understand why their env is not flying.

Reply:
Updated the `README.md` file